config/s3/s3api/s3_object_range_read.yaml

```diff
@@ -0,0 +1,131 @@
+test_1:
+  TEST_ID: TEST-36564
+  object_size:
+    start: 0Kb
+    end: 100Kb
+  result_duration: 2h
+  sessions_per_node: 1
+  range_read: 100bytes
```

Comment:
Range read should have specific range, It would help to repoduce issue if any for specific range.

Reply:
We need to discuss and take new task if required.